script/bin/operator.rs

```diff
@@ -485,9 +485,11 @@ where
 
         self.network_prover
             .prove(&self.pk, &stdin)
-            .strategy(FulfillmentStrategy::Reserved)
+            .strategy(FulfillmentStrategy::Auction)
             .skip_simulation(true)
             .cycle_limit(10_000_000_000)
+            .gas_limit(1_000_000_000)
```

Comment:
I looked at past 20 pages of logs and didn't see any that reached close to 1b gas, the highest I saw was 600m. But just double-checking it will never reach that high (it's important to get reasonable gas limit estimates for provers to estimate if they can do it in time)